cpu/cc2538/periph/uart.c

```diff
@@ -189,30 +129,12 @@ static int init_base(uart_t uart, uint32_t baudrate)
 {
     cc2538_uart_t *u = NULL;
```

Comment:
no need to first assign `NULL` and then assign a different value in the line below... Just write `cc2538_uart_t *u = uart_config[uart].dev;` directly in one line.

Reply:
But also here and in all other UART functions: please add assertions for validating the given parameter(s), as in
```c
assert(uart < UART_NUMOF);
```